app/models/comment.rb

```diff
@@ -264,6 +265,11 @@ def create_first_reaction
     Comments::CreateFirstReactionJob.perform_later(id)
   end
 
+  def after_destroy_actions
+    Users::BustCacheJob.perform_now(user_id)
```

Comment:
Does this need to be inline instead of being done asynchronously?

Reply:
I think there might be a chance that a suer may go to its profile and not see the change because the refresh is on a job and has not be processed.

May be I am a bit pessimistic and we should not block a comment update even if in some rare occasion, user jumps to his profile quickly after update.

I can replace with perform_later. What do you think ?

Reply:
Ok, so I left it unchanged.